ports/servo/main.rs

```diff
@@ -138,11 +140,14 @@ fn main() {
     }
 
     let window = app::create_window(None);
+    //let url = opts::get().url.clone();
 
     // Our wrapper around `Browser` that also implements some
     // callbacks required by the glutin window implementation.
     let mut browser = BrowserWrapper {
-        browser: Browser::new(window.clone()),
+        browser: Browser::new(window.clone(), opts::get().url.clone()
+            .unwrap_or(ServoUrl::parse(PREFS.get("shell.homepage").as_string().unwrap())
+            .unwrap()))
```

Comment:
`opts::get().url` default value should be None, not Some("about:blank") ([see here](https://github.com/sadmansk/servo/blob/95047114eeed77c4c9b310e6d4f04d3a3295a9e2/components/config/opts.rs#L506)).

If `PREFS.get("shell.homepage")` doesn't exist, we should use `"about:blank"` (or exist early).

So you always have a valid URL.